java/src/jmri/implementation/AbstractNamedBean.java

```diff
@@ -2,6 +2,7 @@
 
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import javax.annotation.CheckReturnValue;
+import javax.annotation.OverridingMethodsMustInvokeSuper;
```

Comment:
It's a tricky issue, I think.

I started putting those on because one subclass of AbstractNamedBean had _partially_ implemented the properties handling (with a local `pcs` variable and reimplementation of some but not all of add/drop/fire). That was uncovered when I added some tests that listened to the base properties, and didn't get notified.  

At the same time, it had a reason to reimplement one of them (add, I think).  Maybe another solution could have been found, maybe not, but the fix was to pull the other reimplementations and have that one call the super method as part of its work.  To reduce the odds of that happening again, I added the annotations.

I'm not sure anybody would ever _need_ to reimplement a getter; I can't think of a reason.  But I wouldn't have imagined this case either.

Of course, we control the source, and a `final` can be removed quickly.

So I'm not sure whether this particular pattern is the right answer or not.